modules/mobile/docs/openapi.yaml

```diff
@@ -1286,6 +1286,76 @@ paths:
       security:
         - Bearer: [ ]
       summary: /v0/claims/decision-letters
+  /v0/claims/decision-letters/{document_id}/download:
+    get:
```

Comment:
I think this should also have a params section like:
```
      parameters:
        - description: id of the document being downloaded
          in: path
          name: document_id
          required: true
```

Reply:
Ahh right thanks for catching that. Updated